mobile/src/notifications/notification.epics.ts

```diff
@@ -0,0 +1,75 @@
+/******************************************************************************
+ * Copyright (c) Arcadia, Inc. All rights reserved.
+ ******************************************************************************/
+
+import { ActionsObservable, combineEpics, ofType, StateObservable } from 'redux-observable';
+import { AuthActionType, UserLoggedIn } from '../reducers/auth/auth.action';
+import { catchError, ignoreElements, map, switchMap } from 'rxjs/operators';
+import Push from 'appcenter-push';
+import { EMPTY, from, Subject } from 'rxjs';
+import { Action } from 'redux';
+import { openProfile } from '../navigation/navigation.actions';
+import { AppState, DependenciesContainer } from '../reducers/app.reducer';
+import AppCenter from 'appcenter';
+import { installIdReceived, NotificationAction, NotificationActionType } from './notification.actions';
+
+//----------------------------------------------------------------------------
+const notificationsHandler$ = (action$: ActionsObservable<UserLoggedIn>, state$: StateObservable<AppState>) =>
+    action$.ofType(AuthActionType.userLoggedIn).pipe(
+        switchMap(() => {
+            const notification$ = new Subject<Action>();
```

Comment:
fyi, nothing restricts you from putting this inner code in a named function or in a class for better readability ;) But'll leave that to you guys.

Reply:
It's better to consider this refactoring when notifications handling logic become more complex. This class interface requirements will be clear then.